Revit_Core_Engine/Query/Shape.cs

```diff
@@ -68,6 +70,32 @@ public static Autodesk.Revit.DB.ConnectorProfileType Shape(this MEPCurve mEPCurv
         }
 
         /***************************************************/
+
+        [Description("Determines shape of the MEP Family Symbol based on its primary connector.")]
+        [Input("familySymbol", "Family Symbol to determine the shape of.")]
+        [Input("doc", "Document object of the Family Symbol.")]
+        [Input("settings", "Revit adapter settings.")]
+        [Output("shape", "Shape of an Family Symbol, which can be round, rectangular or oval. If the shape cannot be determined, an invalid connector shape is returned.")]
+        public static ConnectorProfileType Shape(this FamilySymbol familySymbol, Document doc, RevitSettings settings = null)
```

Comment:
Actually, do you need document input here? Can't you take `familySymbol.Document?`

Reply:
Agreed 👍